simulariumio/tests/converters/test_springsalad_converter.py

```diff
@@ -86,12 +86,28 @@
                         "1": {
                             "name": "B",
                             "geometry": {
+                                "displayType": "SPHERE",
                                 "color": "#0080ff",
                             },
                         },
-                        "2": {"name": "GRAY"},
-                        "3": {"name": "CYAN"},
-                        "4": {"name": "BLUE"},
+                        "2": {
+                            "name": "GRAY",
+                            "geometry": {
+                                "displayType": "SPHERE",
+                            },
+                        },
+                        "3": {
+                            "name": "CYAN",
+                            "geometry": {
+                                "displayType": "SPHERE",
+                            },
+                        },
+                        "4": {
+                            "name": "BLUE",
+                            "geometry": {
+                                "displayType": "SPHERE",
+                            },
```

Comment:
I think someone has commented to this effect on previous PRs: but it seems like it would be possible to have a base test object, that you import into all these different test files, and maybe make minor edits to for the specific engine, but this seems like an excessive amount of very similar changes to a lot of files. If you don't want to do it for this PR, it would be good to write up an issue that is reducing repeated code across test files

Reply:
the tox test runner does some weird caching that I don't totally understand, so for identical objects I've tried to do that, but sometimes have run into issues modifying a shared object and then seeing that modification in the next test that uses the hypothetically plain unmodified object. We could add an issue but it's not highest priority imo